jcli/src/jcli_lib/certificate/mod.rs

```diff
@@ -69,8 +69,8 @@ pub enum Error {
     KeyNotFound { index: usize },
     #[error("Invalid input, expected Signed Certificate or just Certificate")]
     ExpectedSignedOrNotCertificate,
-    #[error("Invalid data")]
-    InvalidBech32(#[from] bech32::Error),
+    #[error("Invalid bech32 data")]
```

Comment:
Maybe "invalid Bech32 address string" would be more descriptive.

Reply:
It's not used only for keys though, certificates are bech32 encoded as well while they are stored locally